pkgs/development/python-modules/openapi-python-client/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  pyyaml,
+  typer,
+  autoflake,
+  httpx,
+  python-dateutil,
+  jinja2,
+  black,
+  pydantic,
+  attrs,
+  isort,
+  shellingham,
+}:
+
+buildPythonPackage rec {
+  pname = "openapi_python_client";
+  version = "0.14.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-R/mG9DSUogNSHqAeBfdxL2HUvkHNu9kMJuFcu6G7kLs=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pyyaml
+    typer
+    autoflake
+    httpx
+    python-dateutil
+    jinja2
+    black
+    pydantic
+    attrs
+    isort
+    shellingham
+  ];
+
```

Comment:
```suggestion

  nativeCheckInputs = [
    pytest-mock
    pytestCheckHook
  ];
```

Enable tests here. 

https://github.com/openapi-generators/openapi-python-client/blob/main/pyproject.toml#L39C1-L50

We certainly don't need linters and typing stubs, but probably some pytest plugins, but not -cov.

Reply:
This is also failing when I do that.

Reply:
Post logs :)

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/252042/7d9ac15d-a1f2-4dac-95a2-e597864bc6e1)

Reply:
Here's the result when using sources from Github:

![image](https://github.com/NixOS/nixpkgs/assets/252042/5883e40e-6566-49f9-a77c-fc2a832a422f)

Reply:
That's another leaf package here: https://github.com/openapi-generators/openapi-python-client/blob/main/integration-tests/pyproject.toml

Reply:
Could it be that in the src tarball there are no tests?

Reply:
How are we supposed to include these? In a new derivation file?

Reply:
Yes, it's a different package.

Reply:
Thanks for the tips, I think it's fixed now.